app/server/tests/api-deletePost.test.js

```diff
@@ -0,0 +1,69 @@
+const { AuthFunctions } = require('./functions/authFunctions');
+const { password, name, email } = require('./functions/randomizer');
+const { PostFunctions } = require('./functions/postFunctions');
+const community = require('./functions/communityFunctions');
+
+let auth = new AuthFunctions();
+let post = new PostFunctions();
+
+describe('Testing user\'s ability to delete posts from their communities', () => {
```

Comment:
do actually have to escape that apostrophe?

Reply:
What about double quotes on the outside?

Reply:
That would work, but I don't think that's consistent with the airbnb style guide. 
If there's a strong feeling against escape characters, I would rather go with backticks.